### PR TITLE
Highlighting placeholder of droppable in folder content with color of selected one.

### PR DESCRIPTION
**Pull request by iFlameing**
Fixes https://github.com/plone/volto/issues/1866

### CODE REVIEW

File: theme/themes/pastanaga/extras/contents.less [hunk 1]
```diff
@@ -125,3 +125,15 @@
     background-color: #edf1f2 !important;
   }
 }
+
+#content-core {
```
Comment (tisto) [inline theme/themes/pastanaga/extras/contents.less:129]: @iFlameing this selector is not specific enough.

Comment (iFlameing) [reply theme/themes/pastanaga/extras/contents.less:129]: I pushed the changes.

**Comment by tisto** [comment]
@iFlameing I gave this a shot and I do not see any indication that I am about to drop a file:

Edit-Page (1)

**Comment by tisto** [comment]
@iFlameing actually is seems dropzone supports this feature OOTB: https://www.dropzonejs.com/

**Comment by iFlameing** [comment]
@tisto Please see the issue you are checking the wrong issue. This pr solves this problem. https://github.com/plone/volto/issues/1866

**Comment by iFlameing** [comment]
For dropzone, I think that we don't need a library. We can do that after merging this pr. https://github.com/plone/volto/pull/1821

**Comment by tisto** [comment]
@iFlameing pls rebase.

**Comment by iFlameing** [comment]
ok

**Review by sneridagh — APPROVED** [review]
LGTM

**Pull request merged**